### PR TITLE
Add confirmation dialog box for delete function

### PR DESCRIPTION
**Pull request by LahiruRajapaksha**

### CODE REVIEW

File: components/org.wso2.carbon.siddhi.editor.core/src/main/resources/web/js/design-view/design-grid.js [hunk 1]
```diff
@@ -17,7 +17,7 @@
  */
 define(['require', 'log', 'jquery', 'backbone', 'lodash', 'designViewUtils', 'dropElements', 'dagre', 'edge',
         'windowFilterProjectionQueryInput', 'joinQueryInput', 'patternOrSequenceQueryInput', 'queryOutput',
-        'partitionWith', 'jsonValidator'],
+        'partitionWith', 'jsonValidator', 'bootstrap'],
```
Comment (dnwick) [inline components/org.wso2.carbon.siddhi.editor.core/src/main/resources/web/js/design-view/design-grid.js]: I think we do not need this import "bootstrap" here .

File: components/org.wso2.carbon.siddhi.editor.core/src/main/resources/web/js/design-view/drop-elements.js [hunk 2]
```diff
@@ -17,7 +17,7 @@
  */
 
 define(['require', 'log', 'lodash', 'jquery', 'partition', 'stream', 'query', 'formBuilder', 'aggregation',
-    'jsonValidator', 'sourceOrSinkAnnotation', 'stream', 'table'],
+    'jsonValidator', 'sourceOrSinkAnnotation', 'stream', 'table', 'bootstrap'],
```
Comment (dnwick) [inline components/org.wso2.carbon.siddhi.editor.core/src/main/resources/web/js/design-view/drop-elements.js]: same here we  do we need this bootstrap ?

File: components/org.wso2.carbon.siddhi.editor.core/src/main/resources/web/js/design-view/design-grid.js (hunk 1 above)
Comment (LahiruRajapaksha) [reply components/org.wso2.carbon.siddhi.editor.core/src/main/resources/web/js/design-view/design-grid.js]: Removed the Bootstrap definition from  design-grid.js and drop-element.js

Comment (LahiruRajapaksha) [reply components/org.wso2.carbon.siddhi.editor.core/src/main/resources/web/js/design-view/design-grid.js]: sp1
sp2

**Pull request merged**